### PR TITLE
Additional step to closing, deleting and reopening

### PR DESCRIPTION
**Pull request by StefanieBier**
Please add the additional step required to find the "more information" section on the Case Information panel.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/close-or-delete-case.md [hunk 1]
```diff
@@ -41,7 +41,9 @@ To close a case:
 
 2. On the **Settings** tab, under **Case Information**, click **Select**.
 
-3. Click **Close case**.
+3. At the bottom of the **Case Information** panel, you will see a "..." with more options.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/close-or-delete-case.md:44]:
```suggestion
3. At the bottom of the **Case Information** panel, select **...** (more options).  
```

File: microsoft-365/compliance/close-or-delete-case.md [hunk 2]
```diff
@@ -55,7 +57,9 @@ To reopen a closed case:
 
 2. On the **Settings** tab, under **Case Information**, click **Select**.
 
-3. Click **Reopen case**.
+3. At the bottom of the **Case Information** panel, you will see a "..." with more options.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/close-or-delete-case.md:60]:
```suggestion
3. At the bottom of the **Case Information** panel, select **...** (more options).  
```

File: microsoft-365/compliance/close-or-delete-case.md [hunk 3]
```diff
@@ -82,4 +86,6 @@ To delete a case:
 
 2. On the **Settings** tab, under **Case Information**, click **Select**.
 
-3. Click **Delete case**.
+3. At the bottom of the **Case Information** panel, you will see a "..." with more options.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/close-or-delete-case.md:89]:
```suggestion
3. At the bottom of the **Case Information** panel, select **...** (more options).  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @StefanieBier, please commit my suggested changes if you agree.

**Comment by markjjo** [comment]
@StefanieBier PR merged. Thanks for making this update. I'm going to tweak it a little bit.

**Pull request merged**